### DISCUSSION TITLE
Further Guidance on Negotiated Rate Values

### DISCUSSION THREAD

**Discussion by rlevison-iqvia** [CMSgov/price-transparency-guide#484] [Q&A] [answered]
For the negotiated_type, there are a few ways in which negotiated rates can happen. Allowed values: "negotiated", "derived", and "fee schedule". See additional notes.

***“For negotiated_type, what is the difference between “negotiated” and “fee schedule”?_ 

Additional Comments;
We have review the notes and still have some question regarding the difference between NEGOTIATED & FEE SCHEDULE

- First for fee Schedule, when computing the enrollees cost liability, what costs to consider, (Co-Pay, Co INS, deductible, what costs  are to be considered?
- for Negotiated, we see that this would be the contractually agreed amount for a service, prior to the adjustments.  So why wouldn't Both be required
OR 
What is the difference how do we determine which one to publish?

**Comment by shaselton-usds** [comment, marked as answer]
Hopefully the documentation around the [`negotiated_type`](https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#additional-notes-1) has helped out with these questions.

**Comment by alecstein** [comment]
I appreciate a lot of the thought that went into making this schema. I have a followup question on the meaning of fee schedule that doesn't seem to be addressed. 

My impression was that the only meaning of "fee schedule" is an alternative to a negotiated rate, when the patient's cost liability cannot be determined from the negotiated rate alone. This is like in a payment per-patient-per-month (capitation) situation.

My question is: are insurers using "fee schedule" to mean something besides the above? I see far more "fee schedule" rates than I would expect. In particular, over 90% of the rates I downloaded from one insurer for lab tests were fee schedule rates. I'm wondering if I'm missing some other definition for the term "fee schedule" that's not addressed in the literature.

**Comment by bryanbriegel** [comment]
The TiC rule requires the in-network rate files to make public disclosures of the payer's negotiated rates with providers, and those rates are for the "..purposes of billing, adjudicating, and paying claims.." to those providers. Fee schedule - commonly refers to the Medicare fee schedule (https://www.cms.gov/medicare/medicare-fee-for-service-payment/feeschedulegeninfo)  Where a Fee Schedule is used, whatever % of the fee schedule is agreed to - is the "negotiated rate"

**Reply by alecstein** [reply]
> Where a Fee Schedule is used, whatever % of the fee schedule is agreed to - is the "negotiated rate"

To clarify: do you mean that in certain fee schedule rates, that the "negotiated rate" refers to a percentage, and not a dollar amount?

The TiC rules specifically mention "fee schedule" in the context of capitation agreements, and in this case, the "negotiated rate" is a dollar amount. If my reading of what you're saying is accurate, you're saying that you can have both percentage values and dollar-amount values in the same field, which doesn't align with my understanding of the schema.

**Reply by shaselton-usds** [reply]
I'll let @bryanbriegel correct me if I'm off on his original comment, but it is quite common for negotiated rates to be based off of a percentage of Medicare (or some other fee schedule). Because the negotiated rate is a percentage of a fee schedule, you can still determine the dollar amount for reporting purposes -- a simple case might be "for service X, we've negotiated 150% of Medicare". The reporting would then take whatever the Medicare base-rate is for the service X and multiply it by 150% to get to a dollar amount.

**Reply by alecstein** [reply]
I'm following up with some information from the Federal Register because I'm thinking that not all insurers agree on what "fee schedule" means and it's possible that some insurers are interpreting this to mean different things.

> As the Departments acknowledged earlier in this preamble, negotiated rates, as defined in the final rules, do not always affect a participant's, beneficiary's, or enrollee's cost-sharing liability. To account for alternative reimbursement arrangements such as capitated and bundled payment arrangements, the Departments are renaming the third content element as “in-network rates,” comprised of the following elements, as applicable to the plan's or issuer's payment model: negotiated rate and underlying fee schedule rate, reflected as dollar amounts. Plans and issuers must disclose the underlying fee schedule rate used to determine participant, beneficiary, or enrollee cost-sharing liability only where that rate is different from the negotiated rate. As discussed earlier in this preamble, the final rules require that the cost-sharing liability estimate for a requested covered item or service be calculated using the current underlying fee schedule rate if the plan or issuer uses such a fee schedule. The Departments are of the view that disclosing underlying fee schedule rates will provide the most relevant data on which cost sharing is based, if cost sharing is not based on the negotiated rate, as originally proposed.

In other words, the "fee schedule" rate is what you see when the negotiated rate isn't enough to figure out a patient's cost-sharing liability, as in a capitated agreement, like per-patient per-month.

While we don't have a way of verifying this in the data, what we do see are a few other interpretations of the meaning "fee schedule" by aligning rates:

1. Some insurers use it to mean the Medicare/Medicaid fee schedule. When we align fee schedule rates with Medicare/Medicaid rates, we see that they line up.
2. From a private communication with another person working on this data who has access to insurance fee schedules, sometimes they use "fee schedule" to mean their own internal fee schedule, and the rates align there.
3. Patient cost-liability for capitated agreements (see above).
4. A possible percentage of an existing fee schedule (this is the point that you guys made).

Is it possible that insurers are confused because the words "fee schedule" already apply to an existing concept -- and have not closely read the Federal Register?

Please correct me if I'm wrong. I'm working closely with building an open-source dataset based on these MRFs. You can see examples of our data here https://www.dolthub.com/repositories/dolthub/quest-v3/data/main and our tooling here https://github.com/dolthub/data-analysis/tree/main/transparency-in-coverage/python/mrfutils. Just to show you that my question isn't frivolous. :-)

**Reply by shaselton-usds** [reply]
Hey @alecstein! Thanks for the questions.

For your list, 1-4 are all possibilities of negotiated rates that are based off of fee schedules. "Fee schedule" is not mutually exclusive to any of the scenarios you have listed. It sounds like you're trying to reverse engineer the final, dollar negotiated rate that is based upon a fee schedule to the actual fee schedule itself. While that isn't supported at the moment, what would you envision a schema looking like to support those additional attributes?

**Reply by alecstein** [reply]
@shaselton-usds I'm not opposed to making suggestions actually. But let me try to understand a few things first because your responses have actually set off my alarm bells a little bit.

First of all, a lot of my information about this I got from reading https://www.cms.gov/CCIIO/Resources/Regulations-and-Guidance/Downloads/CMS-Transparency-in-Coverage-9915F.pdf and the discussions on this repo so if there's another resource where these things are discussed, definitely let me know so that I can catch up.

So,  to my question: everything I've read so far about the `negotiated_type = "fee schedule"` suggests that it's a separate arrangement related to capitated insurance agreements. As in what I said here:

> My impression was that the only meaning of "fee schedule" is an alternative to a negotiated rate, when the patient's cost liability cannot be determined from the negotiated rate alone. This is like in a payment per-patient-per-month (capitation) situation.

If the rate is determined by a payer's internal fee schedule (as in, "we've decided we're gonna pay $x for procedure y") that should be labeled as "negotiated" as it reflects how much the provider is going to be getting reimbursed. It shouldn't matter whether the rate comes from some internal fee schedule, Medicare, or otherwise. 

If I'm off on this point, please let me know, since I'm not an expert in fee schedules or anything like that.

What's making me uneasy is that it seems like interpretations of the words "fee schedule" are so flexible that insurers can just throw everything under that umbrella label. And then you lose the ability to discern whether that rate is actually the rate that the provider gets reimbursed, related to some capitation agreement, or something else entirely.

And they're using this term _a lot_. Like, some payers use it exclusively. IMO, you shouldn't be able to have a "fee schedule" rate if you don't also have a "negotiated" rate, at least in most cases.

So, before changing the schema, can we agree that if a) a provider gives some provider group a rate from their internal fee schedule or b) bases that rate off of some other fee schedule, then in these cases, the negotiated type should just be "negotiated"?

And what license do insurance companies have to interpret "fee schedule" in any other way besides the way that's defined in the Transparency in Coverage rules? I have a strong belief that this has been discussed somewhere but I'm having trouble tracking it down.

Thanks for your time.

**Reply by shaselton-usds** [reply]
Just following up on this (and sorry for the delay), you are correct in your understanding of when "fee schedule" should be used for the `negotiated_type`, it should be used in conjunction with either a "negotiated" or "derived" negotiated price object. 

For example, a negotiated rate that is based off of Medicare that where there isn't any other rate in which cost-sharing liability is determined should just be "negotiated". 

From what you've seen, potentially renaming "fee schedule" to reduce confusion might make sense.  As for your previous bullets 1-4, you can still use the value "fee schedule", but again, it would be because the "negotiated"/"derived" rate doesn't account for the patient cost liability in that negotiation.
